cm_sdk/models/alert_config/alert_config.py

```diff
@@ -0,0 +1,84 @@
+from cm_sdk.models import CloudManagerBase, CloudManagerEnum
```

Comment:
You should probably expose these classes via `__all__` in this package's `__init__.py` to reduce nesting.